From 4314b97b476063beb35563c2d61b1ca93eeae08e Mon Sep 17 00:00:00 2001
From: Arpit Agarwal <arp@apache.org>
Date: Mon, 12 Jun 2017 16:03:47 -0700
Subject: [PATCH 2549/2848] HDFS-11967. TestJMXGet fails occasionally.
 Contributed by Arpit Agarwal.

(cherry picked from commit 6ca195997b94abb6eacffcf81ac37c65cf832591)

Change-Id: I5f40e2c1ca35624733cbb229f7fc7d214145f033
---
 .../java/org/apache/hadoop/tools/TestJMXGet.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
index ce98cfd..3e5891d 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
@@ -133,7 +133,7 @@ private static boolean checkPrintAllValues(JMXGet jmx) throws Exception {
     byte[] bytes = null;
     String pattern = "List of all the available keys:";
     PipedOutputStream pipeOut = new PipedOutputStream();
-    PipedInputStream pipeIn = new PipedInputStream(pipeOut);
+    PipedInputStream pipeIn = new PipedInputStream(pipeOut, 1024 * 1024);
     PrintStream oldErr = System.err;
     System.setErr(new PrintStream(pipeOut));
     try {
-- 
1.7.9.5

